meta-ros2-rolling/recipes-bbappends/rosbag2/rosbag2-compression/0001-CMakeLists.txt-drop-dependency-on-zstd_vendor.patch

```diff
@@ -38,12 +38,34 @@ index 176e71ff..e76ab41b 100644
  # Causes the visibility macros to use dllexport rather than dllimport.
  # This is appropriate when building the dll but not consuming it.
  target_compile_definitions(${PROJECT_NAME}_zstd
-@@ -86,7 +86,7 @@ ament_export_include_directories(include)
+@@ -86,7 +86,26 @@ ament_export_include_directories(include)
  ament_export_libraries(${PROJECT_NAME} ${PROJECT_NAME}_zstd)
  ament_export_targets(export_${PROJECT_NAME})
  # order matters here, first vendor, then zstd
 -ament_export_dependencies(pluginlib rcpputils rcutils rosbag2_cpp rosbag2_storage zstd_vendor zstd)
-+ament_export_dependencies(pluginlib rcpputils rcutils rosbag2_cpp rosbag2_storage zstd)
++# don't export zstd as we don't provide zstd from zstd-vendor and rosbag2-{py,transport} would fail with:
++# CMake Error at TOPDIR/tmp-glibc/work/raspberrypi4-webos-linux-gnueabi/rosbag2-py/0.6.0-1-r0/recipe-sysroot/usr/share/rosbag2_compression/cmake/ament_cmake_export_dependencies-extras.cmake:21 (find_package):
++#   By not providing "Findzstd.cmake" in CMAKE_MODULE_PATH this project has
++#   asked CMake to find a package configuration file provided by "zstd", but
++#   CMake did not find one.
++#
++#   Could not find a package configuration file provided by "zstd" with any of
++#   the following names:
++#
++#     zstdConfig.cmake
++#     zstd-config.cmake
++#
++#   Add the installation prefix of "zstd" to CMAKE_PREFIX_PATH or set
++#   "zstd_DIR" to a directory containing one of the above files.  If "zstd"
++#   provides a separate development package or SDK, be sure it has been
++#   installed.
++# Call Stack (most recent call first):
++#   TOPDIR/tmp-glibc/work/raspberrypi4-webos-linux-gnueabi/rosbag2-py/0.6.0-1-r0/recipe-sysroot/usr/share/rosbag2_compression/cmake/rosbag2_compressionConfig.cmake:41 (include)
++#   CMakeLists.txt:21 (find_package)
++ament_export_dependencies(pluginlib rcpputils rcutils rosbag2_cpp rosbag2_storage)
```

Comment:
I don't see zstd here.

Reply:
Sorry, I was confused by GitHub's incorrect colorization of the change.